.github/workflows/ci.yml

```diff
@@ -70,3 +72,31 @@ jobs:
       - run: mix credo --strict --format=oneline
 
       - run: mix test --warnings-as-errors --cover
+
+      - name: Restore PLT cache
+        id: plt_cache_restore
```

Comment:
Do we need to manually do the restore and save? it seems like the action itself will do both already for us as a single action if we want to: [docs](https://github.com/actions/cache?tab=readme-ov-file#restoring-and-saving-cache-using-a-single-action)

Reply:
This is the approach suggested in the dialyxir documentation.

The explanation given is this:


>  # By default, the GitHub Cache action will only save the cache if all steps in the job succeed,
>   # so we separate the cache restore and save steps in case running dialyzer fails.

https://github.com/jeremyjh/dialyxir/blob/master/docs/github_actions.md